### PR TITLE
doc: best practices: pod security standards : improvements

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Better clarification on the doc

### CODE REVIEW

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 1]
```diff
@@ -2,21 +2,26 @@
...
-on the namespace level will **not** be admitted. In this way, it will not be possible to run them.
+Namespace and Pod/Container can be defined with three different policies which are; **Privileged, Baseline and Restricted.** 
+([More info](https://kubernetes.io/docs/concepts/security/pod-security-standards/)). Therefore, Pod(s)/Container(s) that 
+are **not** configured according to the enforced security standards defined globally or
+on the namespace level will **not** be admitted. In this way, it will **not** be possible to run them.
 
-**As a best practice, you must ensure that workloads (Operators and Operands) are defined to run under restricted permissions.**
+**As a best practice, you must ensure that workloads (Operators and Operands) are defined to run under 
+restricted permissions unless they need further privileges. For the cases where Pod/Container(s) requires 
+escalating permissions, the recommendation is to use the label as described bellow**
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:24]: Typo:
```suggestion
escalating permissions, the recommendation is to use the label as described below**
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 2]
```diff
@@ -93,10 +98,12 @@ dep:= &appsv1.Deployment{
 }
 ```
 
-**Note:** For Ansible- and Helm-based Operator projects, your Ansible playbooks or Helm charts must create manifests that comply with the requirements.
+**For Ansible and Helm based language Operators:** Ansible playbooks or Helm charts MUST create manifests that comply
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:101]: Nit:
```suggestion
**For Ansible and Helm language based Operators:** Ansible playbooks or Helm charts MUST create manifests that comply 
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 3]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
 
-**You should ensure the configuration is carried to the Pod/Containers on the bundle CSV (install.spec.deployments.containers).**
+#### How the Operator bundle (CSV) must be configured to apply the standards to the Pod/Containers which are installed by OLM (Operator itself)?
 
-To check an example of CSV which complies with the [restrictive][restricted] policy, see the Golang sample
+For Operators integrated with OLM, they have an Operator bundle with an CSV where the `spec.install.spec.deployments` has a Deployment 
+which defines the Pod/Container(s) those will be installed by OLM to get your Operator itself running on the cluster. 
+Therefore, you will need to check and apply the configurations on it.
+
+**Please, ensure the configuration is carried to the Pod/Containers on the bundle CSV after run make bundle**
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:122]: Nit:
```suggestion
**Note: Ensure the configuration is carried to the Pod/Containers on the bundle CSV after running make bundle**
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 4]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+
+Note that Operator bundle generated with the target is building from looking on 
+the manifest under the `config` directory.
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:125]: Nit:
```suggestion
Note that the Operator bundle generated with the target is built from
the manifests under the `config` directory. 
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 5]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+For Operators integrated with OLM, they have an Operator bundle with an CSV where the `spec.install.spec.deployments` has a Deployment 
+which defines the Pod/Container(s) those will be installed by OLM to get your Operator itself running on the cluster. 
+Therefore, you will need to check and apply the configurations on it.
+
+**Please, ensure the configuration is carried to the Pod/Containers on the bundle CSV after run make bundle**
+
+Note that Operator bundle generated with the target is building from looking on 
+the manifest under the `config` directory. 
+
+To check an example of CSV which complies with the [restricted][restricted] policy, see the Golang sample
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:127]: Nit:
```suggestion
To check an example of a CSV which complies with the [restricted][restricted] policy, see the Golang sample
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 6]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
 under the [testdata/go/v3/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml](https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml)
 
+**Attention** If you need to use the labels to define the Pod/Container(s) installed by OLM (Operator itself) via the CSV are Privileged 
+then, that means set the labels under the `spec.install.spec.deployments`. (_not in the CSV labels on top of the file_)
+
+#### How can I verify my manifest? How to check what are the policy configured on them?
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:133]: Nit:
```suggestion
#### How can I verify my manifest? How do I check what policy is configured?
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 7]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+
+To verify the policy of your Pod/Container(s) you might to use a [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
+cluster as described in the [K8s documentation](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/).
+
+**However, an easy way might be using the tool:** [psachecker](https://github.com/stlaz/psachecker). This tool is only 
+able to be used to check locally the `Deployments/Pods` manifests and not the CSV. Thus, to check out your CSV you can follow the steps:
```
Comment (everettraven) [inline website/content/en/docs/best-practices/pod-security-standards.md:139]: Nit:
```suggestion
able to be used to check locally the `Deployments/Pods` manifests and not the CSV. To check your CSV you can follow these steps:
```

**Review by everettraven — CHANGES_REQUESTED** [review]
Some nits

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 8]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+
+- 1) Create a new `test.yaml` file
+- 2) Add to the file the Deployment schema:
```
Comment (rashmigottipati) [inline website/content/en/docs/best-practices/pod-security-standards.md:142]:
```suggestion
- 2) Add Deployment schema to the file:
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 9]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-csv-deployment-strattegy
```
Comment (rashmigottipati) [inline website/content/en/docs/best-practices/pod-security-standards.md:148]:
```suggestion
  name: test-csv-deployment-strategy
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 10]
```diff
@@ -106,11 +113,51 @@ You might need include this in the installation documentation for your Operator.
...
+spec:
+...
+```
+
+- 3) Now, add to this test the CSV deployment (`spec.install.spec.deployments`)
```
Comment (rashmigottipati) [inline website/content/en/docs/best-practices/pod-security-standards.md:153]:
```suggestion
- 3) Now, add the CSV deployment to this test (`spec.install.spec.deployments`) 
```

**Review by rashmigottipati — APPROVED** [review]
/lgtm

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 11]
```diff
@@ -2,30 +2,32 @@
...
+([More info](https://kubernetes.io/docs/concepts/security/pod-security-standards/)). Therefore, Pod(s)/Container(s) that 
+are **not** configured according to the enforced security standards defined globally or
+on the namespace level will **not** be admitted and it will **not** be possible to run them.
 
-**As a best practice, you must ensure that workloads (Operators and Operands) are defined to run under restricted permissions.**
+**As a best practice, you must ensure that workloads (Operators and Operands) are defined to run under 
+restricted permissions unless they need further privileges. For the cases where Pod/Container(s) requires 
+escalating permissions, the recommendation is to use the label as described below**
 
 #### How should I configure my Operators and Operands to comply with the criteria?
```
Comment (camilamacedo86) [inline website/content/en/docs/best-practices/pod-security-standards.md:25]:
```suggestion
### How should I configure my Operators and Operands to comply with the criteria?
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 12]
```diff
@@ -106,11 +110,51 @@ You might need include this in the installation documentation for your Operator.
     pod-security.kubernetes.io/warn: privileged
 ```
 
-**You should ensure the configuration is carried to the Pod/Containers on the bundle CSV (install.spec.deployments.containers).**
+#### How the Operator bundle (CSV) must be configured to apply the standards to the Pod/Containers which are installed by OLM (Operator itself)?
```
Comment (camilamacedo86) [inline website/content/en/docs/best-practices/pod-security-standards.md:113]:
```suggestion
### How the Operator bundle (CSV) must be configured to apply the standards to the Pod/Containers which are installed by OLM (Operator itself)?
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 13]
```diff
@@ -106,11 +110,51 @@ You might need include this in the installation documentation for your Operator.
...
+
+#### How can I verify my manifest? How do I check what policy is configured?
+
+To verify the policy of your Pod/Container(s) you might to use a [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
+cluster as described in the [K8s documentation](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/).
+
+**However, an easy way might be using the tool:** [psachecker](https://github.com/stlaz/psachecker). This tool is only 
+able to be used to check locally the `Deployments/Pods` manifests and not the CSV. To check your CSV you can follow these steps:
+
+- 1) Create a new `test.yaml` file
```
Comment (camilamacedo86) [inline website/content/en/docs/best-practices/pod-security-standards.md:138]:
```suggestion
- Create a new `test.yaml` file
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 14]
```diff
@@ -106,11 +110,51 @@ You might need include this in the installation documentation for your Operator.
...
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-csv-deployment-strategy
+spec:
+...
+```
+
+- 3) Now, add the CSV deployment to this test (`spec.install.spec.deployments`) 
+- 4) Then, you can run the tool and check if its result will be `restricted` as expected, i.e.,
```
Comment (camilamacedo86) [inline website/content/en/docs/best-practices/pod-security-standards.md:151]:
```suggestion
- Then, you can run the tool and check if its result will be `restricted` as expected (i.e.):
```

**Pull request merged**